### PR TITLE
docs: Restore path to singularity file

### PR DESCRIPTION
**Pull request by tara-det-ai**
## Description

https://hpe-aiatscale.atlassian.net/browse/FE-259

This creates a duplicate of the following:

/setup-cluster/slurm/singularity.html

This should be considered a temporary solution.

### CODE REVIEW

**Comment by jerryharrow** [comment]
Do you want to drop most of the text content so it doesn't show up from a search?

**Review by jerryharrow — APPROVED** [review]
Other than potentially being found via search, this should solve the short-term problem.

File: docs/setup-cluster/deploy-cluster/slurm/singularity.rst [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+ Default Docker Images
+***********************
+
+Each version of Determined utilizes specifically-tagged Docker containers. The image tags referenced
+by default in this version of Determined are described below.
+
++-------------+--------------------------------------------------------------------------+
+| Environment | File Name                                                                |
++=============+==========================================================================+
+| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.12-tf-2.11-cpu-2b7e2a1``    |
```
Comment (jerryharrow) [inline docs/setup-cluster/deploy-cluster/slurm/singularity.rst:29]: We only match parse the following 3 lines my matching the image type in column 1 and getting the path from column 2.   

```
| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.12-tf-2.11-cpu-6eceaca``    |
| NVIDIA GPUs | ``determinedai/environments:cuda-11.3-pytorch-1.12-tf-2.11-gpu-6eceaca`` |
| AMD GPUs    | ``determinedai/environments:rocm-5.0-pytorch-1.10-tf-2.7-rocm-6eceaca``  |
```

Comment (tara-det-ai) [reply docs/setup-cluster/deploy-cluster/slurm/singularity.rst:29]: we may have to manually remove the duplicate result from Algolia

**Pull request merged**